pkg/auth/context_test.go

```diff
@@ -0,0 +1,152 @@
+package auth
+
+import (
+	"testing"
+
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/onsi/gomega"
+)
+
+func TestContext_GetAccountIdFromClaims(t *testing.T) {
+	tests := []struct {
+		name   string
+		claims jwt.MapClaims
+		want   string
+	}{
+		{
+			name:   "Should return empty when tenantUserIdClaim is empty",
+			claims: jwt.MapClaims{},
+			want:   "",
+		},
+		{
+			name: "Should return when tenantUserIdClaim is not empty",
+			claims: jwt.MapClaims{
+				tenantUserIdClaim: "Test_user_id",
+			},
+			want: "Test_user_id",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			gomega.RegisterTestingT(t)
```

Comment:
I think this line can be possibly moved to line just before the start of the for loop (applies to all occurrences in this PR)